application/src/main/java/uk/nhs/hee/tis/revalidation/entity/GmcResponseCode.java

```diff
@@ -36,12 +36,22 @@ public enum GmcResponseCode {
   INVALID_OR_UNEXPECTED_DEFER_DATE("89", "Invalid / Unexpected Defer Date"),
   DOCTOR_NOT_SUBJECT_TO_REVALIDATION("90", "Doctor not subject to revalidation"),
   OPEN_RECOMMENDATION_EXISTS("91", "Open recommendation exists"),
+  DOCTOR_NOT_UNDER_NOTICE("92", "Doctor not under Notice"),
+  DOCTOR_REVALIDATION_ON_HOLD("93", "Doctors revalidation is \"On Hold\""),
+  INVALID_RECOMMENDATION_SUBREASON("94", "Invalid Recommendation Subreason"),
+  RO_NOT_LICENSED("95", "Responsible Officer is not licensed"),
+  MISSING_RO("96", "Missing Responsible Officer"),
+  MISSING_RO_RECOMMENDATION_SUBREASONS("97",
+      "Missing RoRecommendationSubreasons (required when RoRecommendationReason = 1)"),
```

Comment:
May I ask if those errors will be exposed to admin user? I don't think they're all user friendly.

Reply:
Quick look at admins UI seems like they aren't, they'll just be logged in our system (& the method wont crash)

Reply:
Ah actually I was wrong, the message is shown to the user, shall I change this one to "Missing RO Recommendation Subreasons"